style.css

```diff
@@ -0,0 +1,133 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: 'Times New Roman', Times, serif;
```

Comment:
Good job so far by adding some of the changes required by the previous reviewers. 
As you see in the following screenshot, the 'Crete Round' font is not applicated.

<img width="288" alt="image" src="https://user-images.githubusercontent.com/7243132/170587777-2ef67b22-acc1-49e7-8bd6-d6abb8bffc73.png">

To correct it, please move the font-family from the * selector, and put it in the body, like:


```
* {
  padding: 0;
  margin: 0;
  box-sizing: border-box;
}

body {
  font-family: 'Times New Roman', Times, serif;
}
```